src/Sets/VPolygon.jl

```diff
@@ -149,12 +149,15 @@ function remove_redundant_vertices(P::VPolygon;
     return remove_redundant_vertices!(copy(P), algorithm=algorithm)
 end
 
-function linear_map(M::AbstractMatrix, P::VPolygon)
+function linear_map(M::AbstractMatrix, P::VPolygon; convex_hull::Bool=false)
```

Comment:
we have `prune`, `apply_convex_hull`, and now `convex_hull`  boolean options :smile:

Reply:
Oops, you are right. Which one do you prefer?

Reply:
makes sense :+1: